src/ripple/app/misc/Validations.cpp

```diff
@@ -75,7 +75,7 @@ class ValidationsImp : public Validations
 public:
     ValidationsImp (Application& app)
         : app_ (app)
-        , mValidations ("Validations", 128, 600, stopwatch(),
+        , mValidations ("Validations", 4096, 600, stopwatch(),
```

Comment:
`ApplicationImp::setup` calls `mValidations->tune` (line 1008), and sets the size from the config file. It looks like the place to really change this value is in `Config::getSize`, not here.